packages/components/Organisms/Accordion/Accordion.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="rpl-accordion">
     <h2 class="rpl-accordion__title-top" :id="titleId" v-if="title">{{ title }}</h2>
+    <div class="rpl-accordion__collapse">
+      <a class="rpl-accordion__collapse-link" href="" @click.prevent="collapseExpandAll">{{ closeOpenLabel }}</a>
```

Comment:
Hey @rubyllamadora just one minor thing, for better accessibility, this should be a button instead of link. You can find some explanation in `Buttons vs. Links` section in http://web-accessibility.carnegiemuseums.org/content/buttons/

Reply:
Good point. Was using link as I don't want to remove styling in button. Updating this to use `<button>`.

Reply:
> Good point. Was using link as I don't want to remove styling in button. Updating this to use `<button>`.

@rubyllamadora yes, it was a design issue, it's better look like a button instead of link. If we can, may suggest back to designers.